### PR TITLE
fix: reference the types in package json

### PR DESCRIPTION
**Pull request by chelouche9**

### CODE REVIEW

**Comment by chelouche9** [comment]
@xenova added the types to the package json file so the types could be used.

File: package.json [hunk 1]
```diff
@@ -1,8 +1,9 @@
 {
-  "name": "@xenova/transformers",
+  "name": "@chelouche9/transformers",
```
Comment (takoyaro) [inline package.json:2]:
```suggestion
  "name": "@xenova/transformers",
```

Comment (chelouche9) [reply package.json:2]: THank you for noticing

**Comment by xenova** [comment]
Is there a way to do this that allows users to use `npm link`? If I'm not mistaken, this will cause an issue since the types won't be updated?

**Comment by chelouche9** [comment]
> Also, this main.d.ts file won't be updated each time we build.

That is right, the main.d.ts file is not updated. I will try to find a solution that won't require us to change the build process.

**Comment by xenova** [comment]
Changes live on main 👍

**Pull request closed without merging**